docs/architecture/modernize-with-azure-containers/index.md

```diff
@@ -1,19 +1,23 @@
 ---
-title: Modernize Existing .NET Applications With Azure Cloud and Windows Containers (2nd edition)
+title: Modernize Existing .NET Applications With Azure Cloud and Windows Containers (5th edition)
 description: Learn to lift and shift and modernize existing applications to the Azure cloud and containers with this e-book.
-ms.date: 04/28/2018
+ms.date: 12/28/2020
 ---
-# Modernize existing .NET applications with Azure cloud and Windows Containers (2nd edition)
+# Modernize existing .NET applications with Azure cloud and Windows Containers (5th edition)
```

Comment:
I suggest to drop the edition text from the title. We do have this mentioned below anyway.

Reply:
Yes, that makes sense as we are removing edition information from the cover page as well. Updated the title.